airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -91,7 +91,7 @@ def get_link(self, operator: BaseOperator, dttm: datetime):
 
 
 class _BigQueryDbHookMixin:
-    def get_db_hook(self) -> BigQueryHook:
+    def get_db_hook(self: 'BigQueryCheckOperator') -> BigQueryHook:  # type:ignore[misc]
```

Comment:
This is a  nice trick to trade 5 type: ignores for below self.* references with one # type: ignore !